client/state/analytics/actions.js

```diff
@@ -0,0 +1,59 @@
+import curry from 'lodash/curry';
+import isFunction from 'lodash/isFunction';
+import merge from 'lodash/merge';
+
+import {
+	BUMP_STAT,
+	RECORD_EVENT,
+	RECORD_PAGE_VIEW
+} from './constants.js';
```

Comment:
All action types should be declared in `state/action-types.js` ([guideline](https://github.com/Automattic/wp-calypso/blob/master/docs/our-approach-to-data.md#actions)), e.g.:

```js
import {
	ANALYTICS_STAT_BUMP,
	ANALYTICS_EVENT_RECORD,
	ANALYTICS_PAGE_VIEW_RECORD
} from 'state/action-types';
```